dist/livewire.esm.js

```diff
@@ -6816,13 +6820,17 @@ var require_module_cjs8 = __commonJS({
         let toAttributes = Array.from(to.attributes);
         for (let i = domAttributes.length - 1; i >= 0; i--) {
           let name = domAttributes[i].name;
+          if (name === "style")
+            continue;
           if (!to.hasAttribute(name)) {
             from2.removeAttribute(name);
           }
         }
         for (let i = toAttributes.length - 1; i >= 0; i--) {
           let name = toAttributes[i].name;
           let value = toAttributes[i].value;
+          if (name === "style")
+            continue;
```

Comment:
![image](https://github.com/user-attachments/assets/af931b77-3ea3-43e8-b359-7d2d7557b94c)
Degrade with inline styles

Reply:
@calebporzio I think I'm finding the same, that we now can't modify inline styles anymore, based on this.